bentoml/_internal/bento/build_config.py

```diff
@@ -409,17 +423,16 @@ def write_to_bento(self, bento_fs: "FS", build_ctx: str) -> None:
         with bento_fs.open(fs.path.join(py_folder, "version.txt"), "w") as f:
             f.write(f"{version_info.major}.{version_info.minor}.{version_info.micro}")
 
+        # Build BentoML whl from local source if BENTOML_BUNDLE_LOCAL_BUILD=True
+        build_bentoml_editable_wheel(bento_fs.getsyspath(wheels_folder))
```

Comment:
build_bentoml_editable_wheel uses shutil.copytree underneath which requires the target directory to be empty. thus moving it to above the `bento_fs.makedirs(wheels_folder, recreate=True)` line following